executor/insert_common.go

```diff
@@ -530,6 +675,98 @@ func (e *InsertValues) fillRow(ctx context.Context, row []types.Datum, hasValue
 	return row, nil
 }
 
+// fillRowLazy is quite same to fillRow() except it will cache auto increment datum for lazy batch allocation of autoID.
+// This case is used in insert|replace into values (row),(row),(row)...
+func (e *InsertValues) fillRowLazy(ctx context.Context, row []types.Datum, hasValue []bool, rowIdx int) ([]types.Datum, error) {
```

Comment:
This function shares some codes with `fillRow`, and the two functions are with same signature, can we use a flag variable of `InsertValues`(for example, `lazyFillAutoID`) to indicate if the filling logic is lazy, and try to combine two functions?

Reply:
Nice idea！